npm/private/utils.bzl

```diff
@@ -755,4 +752,9 @@ utils = struct(
     exists = _exists,
     replace_npmrc_token_envvar = _replace_npmrc_token_envvar,
     is_tarball_extension = _is_tarball_extension,
+
+    # Only for testing
+    npm_registry_url = _npm_registry_url,
```

Comment:
export via another struct named `test_utils` so that it is enforced by the load statement?

Reply:
I was pondering that. Means the tests will import two things (normal + test-only) but makes this clearer 👍